.taskcat.yml

```diff
@@ -20,6 +23,4 @@ tests:
       QSS3BucketName: $[taskcat_autobucket]
       QSS3KeyPrefix: quickstart-aws-utility-meter-data-analytics-platform/
       QSS3BucketRegion: $[taskcat_current_region]
-    template: ./templates/main.template.yaml
-    regions: 
-    - us-west-2
\ No newline at end of file
+    template: ./templates/main.template.yaml
```

Comment:
Do we not need regions defined, where we want to run this test?

Reply:
No tests will run in all regions defined here https://github.com/aws-quickstart/quickstart-aws-utility-meter-data-analytics-platform/blob/c51063dff0fd0d1b99f3dbbf80f00ad469c34b8c/.taskcat.yml#L7